packaging/linux/post_install.sh

```diff
@@ -39,9 +39,11 @@ if [ -z "$currlink" ] ; then
         if killall kbfsfuse &> /dev/null ; then
             echo Shutting down kbfsfuse...
         fi
-        rmdir "$rootlink"
         echo You must run run_keybase to restore file system access.
     fi
+    if rmdir "$rootlink" &> /dev/null ; then
+        echo Replacing old $rootlink directory.
```

Comment:
People sometimes have files in their old {{/keybase}} if they tried to write there when it wasn't mounted. I guess in that case they'll still hit the bug I saw?

Reply:
Sure, that could happen.  Would you prefer a hard error if the `rmdir` fails?  It's a little tricky because this is during package installation, not during `run_keybase`.

Reply:
Maybe we should do something like "if that directory exists, and `rmdir` fails to remove it here, print a big detailed error message telling the user to delete it manually, and then return an error"?

Reply:
I think your `-T` suggestion below is a good solution for this problem too.  It will leave us without a `/keybase` link, but that seems ok if they already have something there, right?  Not sure it's worth erroring out the package upgrade.

Reply:
`mv /keybase /keybase.old && chown root:root /keybase.old && chmod 0700 /keybase.old` ?

Reply:
(as a fallback?)